x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/scalar/string/Reverse.java

```diff
@@ -93,28 +93,24 @@ public boolean foldable() {
     }
 
     @Evaluator
-    static BytesRef process(BytesRef val, @Fixed Locale locale) {
```

Comment:
If we don't need the Locale, then `Reverse` extending `EsqlConfigurationFunction` is not needed either.

Reply:
Done in https://github.com/elastic/elasticsearch/pull/113297/commits/364d357b6f23766f013eb00230915eaf7f125d31